Makefile

```diff
@@ -43,37 +43,32 @@ setup: ## Setup proposals and generate an initial key
 
 stop-host:  ## 🏃 Stop the host
 	@echo -e "\e[34m$@\e[0m" || true
-	sudo lsof -t -i :8000 | xargs -r sudo kill -9
+	source ./scripts/ccf/sandbox-local/down.sh
```

Comment:
Using CCF only from the container is indeed a choice we need to make carefully, my reasoning for supporting this choice are:
- It mimics how the real system will behave (i.e. Run CCF network somewhere else, and apply KMS code/policies/etc to it)
- It allows us to have a much simpler dev environment as you don't need to work on top of the CCF dev container or install CCF manually
- It makes running against KMS on different styles of CCF network trivial (sandbox locally, sandbox in aci, mCCF while it lasts, azure-cleanroom etc.)

I believe the considerations you raise aren't bad enough to outweigh the benefits for the following reasons:
- Persistent state: Using `docker compose down` cleans up for you, I've never had trouble with it
- Rebuilding: 
  - Since the container image is just the official CCF dev image (i.e. no KMS code), it doesn't typically change while developing the KMS
  - In my experience starting or recreating the container takes ~5 seconds, building the kms code often takes longer

Reply:
It's possible, we would need:
- A local environment with virtual CCF installed and all it's dependencies 
  - See the changes to devcontainer.json, requires developing inside the ccf/dev container or complex setup
- We can then add `/scripts/ccf/sandbox-local-non-container/` directory with up and down scripts 
  - We would need a some extra code to handle the lifecycle stuff which docker compose does for us in this PR

Reply:
Personally I see that as a pain, and I don't see any benefits vs just starting the off the shelf CCF container and treating it like every other use case of CCF

Reply:
I've done several tests including deployment and did not see issues.